llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -191,6 +192,8 @@ static void parseArgs(int argc, char **argv) {
       std::exit(1);
     }
   }
+
+  LoadDwarfCallSites = Args.hasArg(OPT_dwarf_callsites);
```

Comment:
Should we assert that `--callsites-yaml-file` and `--dwarf-callsites` are not both set?

Reply:
While it's unlikely that both will be used, I don't see a reason that you shouldn't be able to use them together.